src/components/EmptyList/EmptyList.tsx

```diff
@@ -3,11 +3,12 @@ import Search from "@material-ui/icons/Search";
 
 interface IEmptyListProps {
     message: string;
+    padding: "diskSpace" | "miningRewards" | "nodeList";
```

Comment:
Why padding is dependent on the graph type? That should definitely be avoided...